src/simsopt/field/magneticfieldclasses.py

```diff
@@ -502,6 +503,181 @@ def from_dict(cls, d, serial_objs_dict, recon_objs):
         return field
 
 
+class DipoleField(MagneticField):
```

Comment:
The magneticfieldclasses.py file is pretty big already, and should probably be split up. What do you think of putting `DipoleField` in its own file?